fluidefi/fluidefi-caspernet-aggregator-tools
# Deploy saver

**Summary of change:**
- Added a new micro-service, deploy saver, which takes a parsed deploy and save it in the datasource. 

**List of tasks:** 
- [x] Define types.
- [x] Define interfaces.
- [x] Define class.
- [x] Create fixture.
- [x] Create mock datasource. 
- [x] Generate the docs. 

**Steps for testing**
- Make sure your local git is up to date
```bash
git pull
```

- Checkout the branch `feature/deploy-saver`
```bash
git checkout feature/deploy-saver
```

- Run the test across the test-net
```bash
npm run test
```

- Confirm deploy-saver test has passed.
- Run the docs using live server and make sure it have been updated.
